src/components/OrderBookChart.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useEffect, useRef } from 'react'
+import BigNumber from 'bignumber.js'
+import styled from 'styled-components'
+
+import * as am4core from '@amcharts/amcharts4/core'
+import * as am4charts from '@amcharts/amcharts4/charts'
+import am4themesSpiritedaway from '@amcharts/amcharts4/themes/spiritedaway'
+
+import { getNetworkFromId, safeTokenName } from 'utils'
+
+import { TokenDetails, Network } from 'types'
+
+export interface OrderBookChartProps {
+  baseToken: TokenDetails
+  quoteToken: TokenDetails
+  networkId: number
+  data: PricePointDetails[]
+}
+
+export const Wrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  /* min-height: 40rem; */
+  /* height: calc(100vh - 30rem); */
+  min-height: calc(100vh - 30rem);
+  text-align: center;
+  width: 100%;
+  height: 100%;
+  min-width: 100%;
+
+  .amcharts-Sprite-group {
+    font-size: 1rem;
+  }
+
+  .amcharts-Container .amcharts-Label {
+    text-transform: uppercase;
+    font-size: 1.2rem;
+  }
+
+  .amcharts-ZoomOutButton-group > .amcharts-RoundedRectangle-group {
+    fill: var(--color-text-active);
+    opacity: 0.6;
+    transition: 0.3s ease-in-out;
+
+    &:hover {
+      opacity: 1;
+    }
+  }
+
+  .amcharts-AxisLabel,
+  .amcharts-CategoryAxis .amcharts-Label-group > .amcharts-Label,
+  .amcharts-ValueAxis-group .amcharts-Label-group > .amcharts-Label {
+    fill: var(--color-text-primary);
+  }
+`
+
+export enum Offer {
```

Comment:
nit: maybe `QuoteType` or  `PriceQuoteType`  (I know the names are not from this PR)

Reply:
Still think it could be mistaken for `Price for Quote Token`